tests/configmap_test.yaml

```
@@ -36,3 +36,16 @@ tests:
       - notMatchRegex:
           path: data.airflow_local_settings\.py
           pattern: '{"somekey":"somevalue"}'
+  - it: should use default auth_backend value deny_all
+    asserts:
+      - matchRegex:
+          path: data.airflow\.cfg
+          pattern: "airflow.api.auth.backend.deny_all"
```

```suggestion
          pattern: "auth_backend: 'airflow.api.auth.backend.deny_all'"
```

If this works, this is probably better.